<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FriendInfo</class>
 <widget class="QDialog" name="FriendInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>好友资料</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>31</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>账号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>头像</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>301</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>年</string>
   </property>
  </widget>
  <widget class="QComboBox" name="DateCombo">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>301</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>271</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>性别</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>个性签名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AgeEdit">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>327</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="SexCombo">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lvLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>76</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="NameEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="FaceButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>72</width>
     <height>68</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>301</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>日</string>
   </property>
  </widget>
  <widget class="QComboBox" name="MonthCombo">
   <property name="geometry">
    <rect>
     <x>182</x>
     <y>301</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>73</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>等级</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SignEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="AccountList">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>257</x>
     <y>301</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>月</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>411</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>所在地</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>生日</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>327</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>年龄</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>383</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>故乡</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>410</y>
     <width>518</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QComboBox" name="LocalP"/>
    </item>
    <item>
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>省</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="LocalC"/>
    </item>
    <item>
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>市</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="LocalL"/>
    </item>
    <item>
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>区</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>371</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/C:/Users/Wang/Desktop/Image/t01eb0ee49bbc2a79ea.jpg</pixmap>
   </property>
  </widget>
  <widget class="QComboBox" name="YearCombo">
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>301</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>355</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>手机号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PhoneEdit">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>355</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>380</y>
     <width>518</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QComboBox" name="HomeP"/>
    </item>
    <item>
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>省</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="HomeC"/>
    </item>
    <item>
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>市</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="HomeL"/>
    </item>
    <item>
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>区</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>昵称</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>0</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/Images/叉 - 副本.png</normaloff>:/Images/叉 - 副本.png</iconset>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
